prac_09/unreliable_car.py

```diff
@@ -0,0 +1,25 @@
+from prac_09.car import Car
+from random import randint
+
+class UnreliableCar(Car):
+
+    def __init__(self, name, fuel, reliability):
+        """Initialise a unreliable car instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        """Drive the car a given distance.
+
+               Drive given distance if car has enough fuel and has a random number
+               higher than distance or drive until fuel runs out return the distance actually driven.
+               """
+        if distance > randint(0, 100):
+            distance = 0
+        elif distance > self.fuel:
```

Comment:
This looks very similar to the "drive" method in car so feels like it isn't DRY.  Is there a way to reuse the parent class method here instead?